content/english/python-pixel/A.Colors/Basics-of-Color/_index.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
Please change the path of the file to content/english/python-pixel/colors/basics-of-colors.md (instead of content/english/python-pixel/A.Colors/Basics-of-Color/_index.md)

Reply:
OK